.betterer.eslint.config.js

```diff
@@ -87,7 +87,13 @@ module.exports = [
               message: 'Use Stack component instead.',
             },
             {
-              group: ['@grafana/ui/src/*', '@grafana/runtime/src/*', '@grafana/data/src/*'],
+              group: [
+                '@grafana/ui/src/*',
+                '@grafana/runtime/src/*',
+                '!@grafana/runtime/src/unstable',
+                '@grafana/data/src/*',
+                '!@grafana/data/src/unstable',
+              ],
```

Comment:
Not sure if we want to exclude `unstable` paths for this rule, so I did it anyway 😉  . Let me know what you think, thanks 🙏

Reply:
Fixed 👍